requirements.txt

```diff
@@ -1,9 +1,187 @@
-celery>=5
-gitpython
-kombu>=5 # A celery dependency but it's directly imported
-packaging
-pyarn
-requests_kerberos
-requests
-semver
-setuptools
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile --generate-hashes --output-file=requirements.txt requirements.in
+#
+amqp==5.0.5 \
```

Comment:
This needs to be pinned to `5.0.2` for now, see the related command in the Dockerfile.
Speaking of, could you remove that command and move the related comment here instead?